### DISCUSSION TITLE
Pressure Trend Discrepancy

### DISCUSSION THREAD

**Discussion by lensherm** [briis/hass-weatherflow2mqtt#156] [Q&A] [answered]
I'm wondering if I potentially misconfigured something, or if there's a different sample size being used to calculate this in here. There's a difference between what the pressure trend status is showing in the Tempest app and their website and what's showing up in HA. For example, as of now, the pressure has been slowly falling for approximately 4 hours and the app, site and HA reflect this, when looking at the graph. However, while the app and the tempestwx.com status have been showing Falling in the trend field, yet the status in HA is showing as Steady most of that time.

**Comment by briis** [comment, marked as answer]
The pressure trend is a calculated value as this is not supplied directly from the station. It compares current value with the value 3 hours ago.
I just looked at my system and it seems to follow the App pretty closely.
The only user supplied value that impacts this value, is the elevation of your station, as that is used to calculate the Sea Level Pressure from the Station Pressure.
